src/site/_data/i18n/courses.yml

```diff
@@ -4,4 +4,9 @@ learn_css:
     es: ¡Aprenda CSS!
   description:
     en: "Let’s gleefully breakdown complex aspects of CSS into digestible episodes covering everything from accessibility to z-index."
-    es: "Dividamos alegremente los aspectos complejos de CSS en episodios digeribles que abarquen todo, desde la accesibilidad hasta el z-índex."
\ No newline at end of file
+    es: "Dividamos alegremente los aspectos complejos de CSS en episodios digeribles que abarquen todo, desde la accesibilidad hasta el z-índex."
+learn_forms:
+  title:
+    en: Learn Forms!
+  description:
+    en: "..."
```

Comment:
Is this a placeholder or literally supposed to be dots?

Reply:
Placeholder. Would you like it to be changed?

Reply:
I don't mind. I guess it is obvious a placeholder but I worry that without a TODO it will be missed :)